src/ohri-form.component.tsx

```diff
@@ -332,6 +332,7 @@ const OHRIForm: React.FC<OHRIFormProps> = ({
                         handlers={handlers}
                         workspaceLayout={workspaceLayout}
                         isSubmitting={isSubmitting}
+                        setIsSubmitting={setIsSubmitting}
```

Comment:
Am curious what the difference between the 2 will be. Don't they do similar things?

Reply:
I am not sure I understand what you are alluding to, kindly explan.